Makefile

```diff
@@ -1,14 +1,22 @@
-OKTA_VERISON = 1.0.8
+.SILENT:
+OKTA_VERISON = 1.0.10
 IMAGE_NAME ?= contino/okta-aws:$(OKTA_VERISON)
 TAG = $(OKTA_VERISON)
 
+.PHONY: build
 build:
 	docker build -t $(IMAGE_NAME) .
 
+.PHONY: test
 test:
-	docker run --rm -it $(IMAGE_NAME) aws --version
-	docker run --rm -it $(IMAGE_NAME) make --version
+	docker run --rm -it --entrypoint="aws" $(IMAGE_NAME) --version
+	docker run --rm -it --entrypoint="make" $(IMAGE_NAME) --version
```

Comment:
Ha! Yeah, it's annoying how they've set an explicit `entrypoint` now, so we need this extra guff to do what we want 😞